portality/dao.py

```diff
@@ -92,6 +92,10 @@ def created_timestamp(self):
     def last_updated(self):
         return self.data.get("last_updated")
 
+    @property
+    def last_updated_timestamp(self):
+        return datetime.strptime(self.last_updated, "%Y-%m-%dT%H:%M:%SZ")
```

Comment:
do we use the ```portality.lib.dates``` module for specifying our preferred date stamp format, or perhaps reading it straight from ```app.config.get("DEFAULT_DATE_FORMAT")```?

I don't know if we've done this before, but we must have quite a few formatter strings dotted around the code that are supposed to be the same.

Reply:
Yeah, we should do.  Probably a concerted effort to move over to `dates` is required.  Let me ponder it, and maybe I'll make the change here, and perhaps to the rest of this file for this PR.

Reply:
Ok, I've had a think about this, and I'm not going to change it for the PR, as it's an orthogonal question.  Instead I'm going to open an issue to track the idea that we need to move to more standardisation in how we handle dates.